modules/getting-started/con-techdocs-config-cicd.adoc

```diff
@@ -0,0 +1,27 @@
+[id="con-techdocs-config-cicd_{context}"]
+
+= Configuring CI/CD to generate and publish TecDocs sites
+
+TechDocs reads the static generated documentation files from a cloud storage bucket, such as OpenShift Data Foundation. The documentation site is generated on the CI/CD workflow associated with the repository containing the documentation files. You can generate docs on CI and publish to a cloud storage using the `techdocs-cli` tool.
+
+You can use the following example to create a script for TechDocs publication:
+
+[source]
+----
+# Prepare
+REPOSITORY_URL='https://github.com/org/repo'
+git clone $REPOSITORY_URL
+cd repo
+
+# Install @techdocs/cli, mkdocs and mkdocs plugins
+npm install -g @techdocs/cli
+pip install "mkdocs-techdocs-core==1.*"
+
+# Generate
+techdocs-cli generate --no-docker
+
+# Publish
+techdocs-cli publish --publisher-type awsS3 --storage-name <bucket/container> --entity <Namespace/Kind/Name>
+----
+
+The TechDocs workflow starts the CI when a user makes changes in the repository containing the documentation files. You can configure the workflow to start only when files inside the `docs/` directory or `mkdocs.yml` are changed.
```

Comment:
Do we show users how to configure the workflow this way? It would be good to include that here.